wrap

```diff
@@ -0,0 +1,2 @@
+\usepackage{fvextra}
+\DefineVerbatimEnvironment{Highlighting}{Verbatim}{breaklines,breakanywhere,commandchars=\\\{\}}
```

Comment:
It's useless to add a header file it's already in the template

https://github.com/Wandmalfarbe/pandoc-latex-template/blob/30cfc57a0c64fad1202aa1422e2ed3a77535bf02/eisvogel.tex#L348-L356

Only `breakanywhere` option is not used.